setup.py

```diff
@@ -16,7 +16,7 @@ def package_data(pkg, root):
 
 setup(
     name='xblock-pdf',
-    version='1.0.1',
+    version='1.0.3',
```

Comment:
Why 1.0.3 and not 1.0.2?

Reply:
There's already a release that is tagged as [`1.0.2`](https://github.com/open-craft/xblock-pdf/releases/tag/v1.0.2).

Unfortunately, the tag and the release didn't match :slightly_frowning_face: